class/person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permissions: true)
+    @name = name
+    @age = age
+    @parent_permissions = parent_permissions
+  end
```

Comment:
- [ ] You have an attribute id but it is not set to any value when you are initializing it. I would advise you to set it to a random number `@id = Random.rand(1..1000)` in the initialize method. 👍